components/admin_console/billing/billing_subscriptions/index.tsx

```diff
@@ -64,7 +64,7 @@ const BillingSubscriptions: React.FC<Props> = () => {
 
     let isFreeTrial = false;
     let daysLeftOnTrial = 0;
-    if (subscription && subscription.trial_end_at > 0 && subscription.status === 'trialing') {
+    if (isCloud && subscription && subscription.is_free_trial === 'true') {
```

Comment:
I think `isCloud` is not needed at this point given that the subscription screen does not show in non-cloud and is already checked by the System Console

Reply:
ok